custom_components/versatile_thermostat/underlyings.py

```diff
@@ -1250,11 +1251,20 @@ async def start_cycle(
         # if force:
         await self.set_valve_open_percent()
 
+    @overrides
+    async def turn_off(self):
+        _LOGGER.debug("%s - ValveRegulation turn_off -> enforce min_opening_degree", self)
+        self._percent_open = 0
+        await self.send_percent_open()
```

Comment:
Send only if the valve is > min to avoid transmitting values when not necessary and save some battery.

Is the upper class turn_off not right ?

```
    async def turn_off(self):
        """Turn heater toggleable device off."""
        _LOGGER.debug("%s - Stopping underlying valve entity %s", self, self._entity_id)
        # Issue 341
        is_active = self.is_device_active
        self._percent_open = self.cap_sent_value(0)
        if is_active:
            await self.send_percent_open()
```